00_Base/package.json

```diff
@@ -34,7 +35,7 @@
     "class-transformer": "^0.5.1",
     "fastify": "^4.22.2",
     "reflect-metadata": "^0.1.13",
-    "tslog": "^4.9.2",
```

Comment:
if this change is important enough to include, should we propagate it to the other package.json files?

Reply:
so this was one of the fixes we did last week for tslog and tsyringe... I just ended up copying and pasting the whole file and this was included... I can revert this one if you would like or propagate it to other packages... maybe we can just remove all carrots and tildas in one separate PR and I can just revert this here?

Reply:
yes, please revert it so that it can have its own pr.

Reply:
reverted